### ISSUE TITLE
Required redundant fields to generate fiber photometry session metadata

### ISSUE THREAD

**Issue by XX-Yin** [AllenNeuralDynamics/aind-data-schema#920]
**Describe the bug**
`fiber_connections` are required to generate the session metadata stream (`Stream`) with modality `Modality.FIB`. It's better to just store these connections in the rig metadata. And set this `fiber_connections` to optional during generating session metadata. 

Keeping @hagikent updated as he is the leading person using fiber optic photometry.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by hagikent** [comment]
We discussed this and concluded that it's nonsense to have those info, that could be different across sessions, in the `rig.json`, which is static.
`name` should be inherited from `rig.json` but other parameters have to be de novo inserted into `sessions.json` by the GUI session by session.

**Comment by saskiad** [comment]
@hagikent I understand your comment to mean we should not make these changes and close this issue, correct?

**Comment by XX-Yin** [comment]
> We discussed this and concluded that it's nonsense to have those info, that could be different across sessions, in the `rig.json`, which is static. `name` should be inherited from `rig.json` but other parameters have to be de novo inserted into `sessions.json` by the GUI session by session.

This is also what I thought. We put these static fields to the rig.json and only reference names in the session.json. However, `light_sources`, `detectors` and `fiber_connections` `configs` are required to generate `Stream` with modality `Modality.FIB`. 

![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/109394934/c9345f28-1c37-41cc-9b48-4f827bba769f)

**Comment by hagikent** [comment]
@saskiad yes, for 'fiber_connections', I don't see a reason to modify it for now.

@XX-Yin I'm not fully sure what you mean here... as `fiber_connections` contains `power`, which is not static, we can not describe it in the `rig`.

`name` of those should be extracted from `rig` for correspondence. Other entries (mostly experimental parameters) should be provided when generating the sessions. 

For example, for `fiber_connections`:

fiber_connections=[
FiberConnectionConfig(
patch_cord_name="**_this should be inherited from rig_**",
patch_cord_output_power=20,  **_<-provided per each session_** 
output_power_unit="microwatt", **_<-can be hardcoded_**
fiber_name="**_this should be inherited from rig_**",
),

Let me know if still unclear.

**Comment by XX-Yin** [comment]
Yes, we should keep the fiber connection part. Do we want to keep `light_sources` and `detectors` as shown below? 
![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/109394934/18443711-160a-4a6c-9f08-cd97b4d08d0e)
Or we can only keep names as shown below. 
![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/109394934/634e62c6-2a27-4418-ade9-6b9157049a32)

For me, either way is ok. We don't need to change anything. I agree to close this issue.
